onadata/apps/api/viewsets/user_profile_viewset.py

```diff
@@ -228,6 +246,17 @@ def create(self, request, *args, **kwargs):
         profile = response.data
         user_name = profile.get("username")
         cache.set(f"{USER_PROFILE_PREFIX}{user_name}", profile)
+
+        # send notification on creating new user account
+        user = User.objects.get(username__iexact=user_name)
+        send_message(
+            instance_id=user_name,
+            target_id=user.id,
+            target_type=USER,
+            user=user,
+            message_verb=USER_CREATED,
+        )
```

Comment:
Could we move this directly into the `UserProfileSerializer` class instead? I think the above additional query for the user will be unnecessary in such a case.